internal/pkg/runtime/engines/singularity/config/config.go

```diff
@@ -90,6 +90,7 @@ type JSONConfig struct {
 	TargetUID     int           `json:"targetUID,omitempty"`
 	TargetGID     []int         `json:"targetGID,omitempty"`
 	LibrariesPath []string      `json:"librariesPath,omitempty"`
+	DeleteImage   bool          `json:"deleteImage,omitempty"`
```

Comment:
With this option, doesn't this render it impossible for two executions of singularity to share the same temporary splatted fs?

Reply:
each execution has it's own copy